blockchain/genesis/genesis.go

```diff
@@ -55,16 +55,16 @@ func defaultConfig() Genesis {
 			DardanellesBlockHeight:  1816201,
 			DaytonaBlockHeight:      3238921,
 			EasterBlockHeight:       4478761,
-			FbkMigrationBlockHeight: 4330441,
-			FairbankBlockHeight:     4339081,
+			FbkMigrationBlockHeight: 5159881,
+			FairbankBlockHeight:     5165641,
 		},
 		Account: Account{
 			InitBalanceMap: make(map[string]string),
 		},
 		Poll: Poll{
 			PollMode:                         "nativeMix",
 			EnableGravityChainVoting:         true,
-			GravityChainCeilingHeight:        19937937, // TODO: replace with accurate height before Fairbank launch
+			GravityChainCeilingHeight:        10199000,
```

Comment:
average ether block time is 13second: https://etherscan.io/chart/blocktime
which gives 10193315 at June 02 2pm

use 12.5 second block time --> 10199000 for some extra margin